api/models/User.js

```diff
@@ -17,7 +18,9 @@ module.exports = bookshelf.Model.extend(merge({
   comments: function () {
     return this.hasMany(Comment)
     .query(q => {
-      q.where('posts.user_id', 'NOT IN', BlockedUser.blockedFor(this.id))
+      // TODO: this breaks recent activity, but it is sketchy to take out here.
```

Comment:
I'm still wanting to get back to this. I missed your point twice on this but now clearly get it. Maybe the solution will be to `Comment.find({ user_id: xxx})` including the User relation where this is otherwise used now. I don't know, but bookmarked. I've created a backlog item for this.